apriltag/src/main/native/thirdparty/apriltag/src/apriltag.c

```diff
@@ -1053,9 +1056,10 @@ zarray_t *apriltag_detector_detect(apriltag_detector_t *td, image_u8_t *im_orig)
                 // SHARPEN the image by subtracting the low frequency components.
                 image_u8_t *orig = image_u8_copy(quad_im);
                 image_u8_gaussian_blur(quad_im, sigma, ksz);
-
-                for (int y = 0; y < orig->height; y++) {
-                    for (int x = 0; x < orig->width; x++) {
+                
+                //TO-DO add check of roiX and roiY
```

Comment:
Are these going to be done in this PR? If so, this PR should be marked as a [draft](https://github.blog/news-insights/product-news/introducing-draft-pull-requests/).

Also, IDEs typically highlight `TODO` instead of `TO-DO`.

A space should be included between `//` and its content.

Reply:
Excuse me I didn't quite understand what you want with this one, I just marked it for me to remember, I can delete it if that's a problem but I intent on making it part of the PR

Reply:
Since you still have more to add, I'll convert this PR to a draft.